backend/src/controller/auctions.ts

```diff
@@ -129,9 +128,10 @@ export const getAllAuctions = (req: Request, res: Response) => {
 
         try {
             auctions
-                .find({ startDate: { $gt: minDate } })
-                .find({ price: { $gt: priceMin } })
-                .find({ price: { $lt: priceMax } })
+                .find()
```

Comment:
question: is chaining `find()` methods the same as passing the filtering options once?